tests/test_ops/test_tensorrt.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+import numpy as np
+import onnx
+import pytest
+import torch
+
+onnx_file = 'tmp.onnx'
+trt_file = 'tmp.engine'
+
+
+@pytest.mark.skipif(
+    not torch.cuda.is_available(), reason='CUDA is required for test_roialign')
+def test_roialign():
+    try:
+        from mmcv.tensorrt import (TRTWraper, onnx2trt, save_trt_engine,
+                                   is_tensorrt_plugin_loaded)
+        if not is_tensorrt_plugin_loaded():
+            pytest.skip('test requires to complie TensorRT plugins in mmcv')
+    except (ImportError, ModuleNotFoundError):
+        pytest.skip('test requires to install TensorRT from source.')
+
+    try:
+        from mmcv.ops import RoIAlign
+    except (ImportError, ModuleNotFoundError):
+        pytest.skip('test requires compilation')
+
+    # trt config
+    fp16_mode = False
```

Comment:
Is fp16 supported yet?

Reply:
This plugin does not support fp16 for now. If the flag is turned on. Other layers (besides plugin) of the model will work in fp16 mode.